src/mondeath.cpp

```diff
@@ -57,42 +57,33 @@ void mdeath::normal(monster *z)
     if ( z->type->in_species( ZOMBIE )) {
             sfx::play_variant_sound( "mon_death", "zombie_death", sfx::get_heard_volume(z->pos()));
         }
-    m_size monSize = z->type->size;
+    bool bigBeast = z->type->size >= MS_MEDIUM;
```

Comment:
Some old code uses `camelCase`, but separating words with an underscore is preferred.